app/Http/Controllers/ClassController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Models\classe;
+
+use function Pest\Laravel\post;
+
+class ClassController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     */
+    public function create()
+    {
+        return view('add_classe');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     */
+    public function store(Request $request)
+    {
+        $classname= $request->input('classname');
```

Comment:
you can use a little easier syntax

rather than 
```php
$classname= $request->input('classname');
```

use
```php
$classname= $request->classname;
```

Reply:
ok